README.md

```diff
@@ -174,12 +174,13 @@ Options:
                                                  Preview TFM. See
                                                  https://dotnet.microsoft.com/platform/support/policy/dotnet-core for
                                                  details for what these mean.
+                                                                    --format                                        Specify what format desired. Currently supports html and sarif. 
   --version                                      Show version information
   -?, -h, --help                                 Show help and usage information
 ```
 The output of the analyze command is a report in SARIF format. SARIF is based on JSON and can be viewed using the following viewers:
 
-- Any text editor. 
+- Any text editor.
 - [VS extension for SARIF Viewer](https://marketplace.visualstudio.com/items?itemName=WDGIS.MicrosoftSarifViewer) for a richer experience.
 
 Sample of the report in VS SARIF Viewer : ![Analysis Report](/docs/images/AnalysisReport.PNG)
```

Comment:
Could you add details about HTML Viewer here? similar to how we are taking about SARIF log, with a screenshot of a sample HTML View.